### ISSUE TITLE
Change Outline Color

### ISSUE THREAD

**Issue by shahdivyank** [acm-ucr/membership-portal#314]
<img width="464" alt="image" src="https://user-images.githubusercontent.com/36285610/234437811-71e8cc0f-07ea-42a9-b7fd-adc16ca3bba0.png">

- Change focus outline color to be black instead of blue
- You can also play around with purple to see which one looks better
